.ci/configure_crio_for_kata.sh

```diff
@@ -18,12 +18,24 @@ if [ "$ID" == "centos" ]; then
 fi
 
 crio_config_file="/etc/crio/crio.conf"
+crio_config_dir="/etc/crio/crio.conf.d"
 runc_flag="\/usr\/local\/bin\/crio-runc"
 kata_flag="\/usr\/local\/bin\/kata-runtime"
 
 minor_crio_version=$(crio --version | egrep -o "[0-9]+\.[0-9]+\.[0-9]+" | head -1 | cut -d '.' -f2)
 
-if [ "$minor_crio_version" -ge "12" ]; then
+if [ "$minor_crio_version" -ge "18" ]; then
+	echo "Configure runtimes map for RuntimeClass feature with drop-in configs"
+	echo "- Set kata as default runtime"
+	cat <<EOF >> "$crio_config_dir/00-runtime.conf"
```

Comment:
Hi @haircommander ! Just double checking that `00-` is a good number since that any other drop-in in this folder will overwrite Kata's.

Reply:
for some reason I forgot the order :upside_down_face: updated to 99, because I wished for it to overwrite any other configuration